prow/repoowners/repoowners.go

```diff
@@ -409,6 +409,11 @@ func (o *RepoOwners) FindLabelsForPath(path string) sets.String {
 	return entriesForPath(path, o.labels, o.options, false, o.enableMDYAML)
 }
 
+// IsNoParentOwners checks if an OWNERS file path refers to an OWNERS file with NoParentOwners enabled.
```

Comment:
Does this need to check if the key is present?

Reply:
If the key isn't present an empty `dirOptions` is returned and the `NoParentOwners` field will default correctly to false.

Reply:
right. 🙃 LGTM